Zuly-Montaño/tarea-1/index.css

```diff
@@ -0,0 +1,114 @@
+.container{
+    background-image: url(img/fondo.jpg);
+    background-size: cover;
+ 
+    border-radius: 15px;
+    box-shadow: 0px 4px 10px 0px #000000;
+    margin:0px auto;
+    width: 50%;
+    background-color: rgba(0, 255, 255, 0.437);
+    padding: 12px;
+    height: 98vh;
+
+}
+.titulo-flex{
+    background-color: rgba(255, 255, 255, 0.176);
+    display: flex;
+    justify-content: flex-start;
+    margin-right: 60%;
+
+}
+
+
+.Description p {
+    font: oblique bold 95% cursive;
+    color:#000000;
+    text-align: left;
+    height: 40;
+    width: 90;
+    margin-right: 65%;
+    margin-top: 10%;
+    
+    
+    
+}
+
+.text p {
+    font:oblique bold 95% cursive;;
+    color: #000000;
+    text-align: left;
+    margin-right: 60%;
+}
+
+.containerImagen{
+    width:70px;
+    height:70px;
+
+}
+
+.containerImagen img{
+    width:85%;
+    height:80%;
+    float:right;
+    box-shadow: 0px 8px 12px 0px #000000;
+    padding: 3px;
+    
+
+}
+
+.text {
+    margin-top: 42%;
+    display:flex;
+    flex-direction: row-reverse;
+    align-items:center;
+}
+
+.customs  {
+    text-align: left;
+    width: -90px;
+    height: -8px;
+    
```

Comment:
@zuly7 , you have some format issues like empty lines, please use prettier with VS code